src/lambda_codebase/account_processing/tests/test_delete_default_vpc.py

```diff
@@ -0,0 +1,44 @@
+"""
+Tests the delete_default_vpc lambda
+"""
+
+import unittest
+from unittest.mock import MagicMock, patch
+from delete_default_vpc import find_default_vpc
+from botocore.exceptions import ClientError
+
+
+class TestFindDefaultVPC(unittest.TestCase):
+
+    @patch('delete_default_vpc.patch_all')
```

Comment:
Could you mock tenacity's sleep function to speed the tests up?
Like so:

```suggestion
    @patch("tenacity.nap.time.sleep", MagicMock())
    @patch('delete_default_vpc.patch_all')
```